wp-config.php

```diff
@@ -77,11 +77,6 @@
         define( 'WP_TEMP_DIR', sprintf( '/srv/bindings/%s/tmp', PANTHEON_BINDING ) );
     endif;
 
-    // FS writes aren't permitted in test or live, so we should let WordPress know to disable relevant UI
-    if ( in_array( $_ENV['PANTHEON_ENVIRONMENT'], array( 'test', 'live' ) ) && ! defined( 'DISALLOW_FILE_MODS' ) ) :
-      define( 'DISALLOW_FILE_MODS', true );
-    endif;
```

Comment:
I don't think this should be removed. WordPress should still be blocked from modifying the filesystem on non-dev environments.